devel/ecs-cli/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+
+name                ecs-cli
+version             1.2.0
+
+categories          devel
+license             Apache-2
+maintainers         @emcrisostomo openmaintainer
+platforms           darwin
+supported_archs     x86_64
+
+description         A custom Amazon ECS CLI that eases up the cluster setup \
+                    process.
+long_description    \
+                    The Amazon ECS Command Line Interface (CLI) is a command \
+                    line interface for Amazon Elastic Container Service \
+                    (Amazon ECS) that provides high-level commands to simplify \
+                    creating, updating, and monitoring clusters and tasks from \
+                    a local development environment.  The Amazon ECS CLI \
+                    supports Docker Compose, a popular open-source tool for \
+                    defining and running multi-container applications. Use the \
+                    CLI as part of your everyday development and testing cycle \
+                    as an alternative to the AWS Management Console.
+
+homepage            https://github.com/aws/amazon-ecs-cli
+
+master_sites        https://s3.amazonaws.com/amazon-ecs-cli/
+distname            ${name}-darwin-amd64-v${version}
+
+# Amazon provides an uncompressed binary file
+extract.suffix
+
+checksums           rmd160 3b32f61f2ca80264a6a87497137bce1f171a2baa \
+                    sha256 84e1455bf7e46d24f4feaf7d77c55b7b8c0d2fca9b471928833add0f37a4f34a
+
+use_configure       no
+
+# The default build is a binary install of the executable.
+
+extract {
+}
```

Comment:
If you don't want to extract any files, clear `extract.only` (i.e. write `extract.only` on a line by itself, with no value after it), rather than overriding the extract phase.

Reply:
Thanks @ryandesign